src/components/academy/grading/GradingWorkspace.tsx

```diff
@@ -104,6 +114,33 @@ class GradingWorkspace extends React.Component<GradingWorkspaceProps> {
     )
   }
 
+  /**
+   * Checks if there is a need to reset the workspace, then executes
+   * a dispatch (in the props) if needed.
+   *
+   * @param props the props passed to the component
```

Comment:
Not a useful param tag can it be removed? Description at line 118 is fine, though.